tinkernet/node/src/command.rs

```diff
@@ -212,8 +212,7 @@ pub fn run() -> Result<()> {
                 )),
                 #[cfg(feature = "runtime-benchmarks")]
                 BenchmarkCmd::Storage(cmd) => runner.sync_run(|config| {
-                    let partials =
-                        new_partial::<_>(&config, crate::service::parachain_build_import_queue)?;
+                    let partials = new_partial(&config)?;
```

Comment:
How's this related to the PR?

Reply:
had to be removed to run the benchmarks [`pub fn new_partial(config: &Configuration) -> Result<Service, sc_service::Error> {`](https://github.com/InvArch/InvArch-Node/blob/94580c6abbcdeb9c2815c5a681e2fd0b30ac526a/tinkernet/node/src/service.rs#L96)